app/templates/_brief_applications_stats.html

```diff
@@ -1,22 +1,49 @@
-{% import "toolkit/statistics.html" as statistics %}
+<div class="govuk-grid-column-one-third" class="app-stat-block" id="incomplete-applications">
```

Comment:
Chrome is ignoring the second class for me

```suggestion
<div class="govuk-grid-column-one-third app-stat-block" id="incomplete-applications">
```

Reply:
Ugh, see earlier comment re: Brain being mush the day I worked on this. Thanks for the spot!